e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -0,0 +1,149 @@
+*/
+
+function checkStepOne() {
+    cy.get('#tutorialIntroContent').should('be.visible').should('contain', 'Welcome to:');
+    cy.get('#tutorialIntroContent').should('be.visible').should('contain', 'Mattermost');
+    cy.get('#tutorialIntroContent').should('be.visible').should('contain', 'Your team communication all in one place, instantly searchable and available anywhere.');
+    cy.get('#tutorialIntroContent').should('be.visible').should('contain', 'Keep your team connected to help them achieve what matters most.');
+    cy.get('#tutorialIntroCircle0').should('be.visible');
+    cy.get('#tutorialIntroCircle1').should('be.visible');
+    cy.get('#tutorialIntroCircle2').should('be.visible');
+    cy.get('#tutorialNextButton').should('be.visible');
+    cy.get('#tutorialSkipLink').should('be.visible');
+} // check Step One
+
+/*  This function checks the contents of the second step of the tutorial.
+*/
+
+function checkStepTwo() {
+    cy.get('#tutorialIntroTwo').should('be.visible').should('contain', 'Communication happens in public discussion channels, private channels and direct messages.');
+    cy.get('#tutorialIntroTwo').should('be.visible').should('contain', 'Everything is archived and searchable from any web-enabled desktop, laptop or phone.');
+    cy.get('#tutorialIntroTwo').should('be.visible').should('contain', 'Install the apps for PC, Mac, iOS and Android for easy access and notifications on the go.');
+    cy.get('#appDownloadLink').should('be.visible').should('contain', 'PC, Mac, iOS and Android');
+    cy.get('#tutorialIntroCircle0').should('be.visible');
+    cy.get('#tutorialIntroCircle1').should('be.visible');
+    cy.get('#tutorialIntroCircle2').should('be.visible');
+    cy.get('#tutorialNextButton').should('be.visible');
+    cy.get('#tutorialSkipLink').should('be.visible');
+} // check step two
+
+/*  This function checks the contents of the third step of the tutorial.
+*/
+
+function checkStepThree() {
+    cy.get('#tutorialIntroThree').should('be.visible').should('contain', 'Invite teammates when you\'re ready.');
+    cy.get('#tutorialIntroThree').should('be.visible').should('contain', 'Need anything, just email us at feedback@mattermost.com.');
+    cy.get('#tutorialIntroThree').should('be.visible').should('contain', 'Click "Next" to enter Town Square. This is the first channel teammates see when they sign up. Use it for posting updates everyone needs to know.');
+    cy.get('#tutorialIntroCircle0').should('be.visible');
+    cy.get('#tutorialIntroCircle1').should('be.visible');
+    cy.get('#tutorialIntroCircle2').should('be.visible');
+    cy.get('#tutorialNextButton').should('be.visible');
+    cy.get('#tutorialSkipLink').should('be.visible');
+} // check step three
+
+/*  This function checks to see if the Off-Topic channel displays by the header info and the post box.
+*/
+
+function checkOffTopicChannel() {
+    cy.get('#channelHeaderInfo').should('be.visible').should('contain', 'Off-Topic');
```

Comment:
Nitpick: Additional chained assertions should use `and` vs  another `should`.

Reply:
Replaced subsequent shoulds with ands.